mautic/mautic
# Campaign forms source - add contact just to active campaign

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



If you use forms as campaign source, contacts are added to inactive campaigns. This PR fixed it

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create campaign form
3. Create two campaigns
- Published
- Unpublished
4. Submit form
5. See contact was added to both campaign
6. We expect add contact just to active campaigns



<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11130"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
